test/cypress/integration/license_spec.ts

```diff
@@ -122,5 +122,10 @@ describe('License Management', () => {
                 uploadLicense(license.file).then(() => verifyError(license.error));
             });
         });
+
+        it('shows inactive license error', () => {
```

Comment:
Optional:

Maybe to be aligned with the other test case names
```suggestion
        it('handles inactive license error', () => {
```